Forge.Forms/src/Forge.Forms/DynamicExpressions/MultiValueConverters/MultiplyMultiConverter.cs

```diff
@@ -0,0 +1,188 @@
+﻿using System;
+using System.Globalization;
+using System.Windows;
+using System.Windows.Data;
+
+namespace Forge.Forms.DynamicExpressions.MultiValueConverters
+{
+    public class MultiplyMultiConverter : IMultiValueConverter
+    {
+        public object Convert(object[] values, Type targetType, object parameter, CultureInfo culture)
+        {
+            if (values.Length == 2 &&
+                values[0] != null &&
+                values[1] != null)
+            {
+                if (values[0] is decimal dc1)
+                {
+                    if (values[1] is decimal dc2)
```

Comment:
Considering all this chunk of code is being repeated it's a good candidate to refactor into a helper.